### PR TITLE
Add progress page for cluster deployment

### PR DESCRIPTION
**Pull request by rawagner**
Progress messages were tweaked a bit, see code
Screenshot from 2021-09-16 14-29-32

Screenshot from 2021-09-16 15-04-34

### CODE REVIEW

**Comment by matthewcarleton** [comment]
nice progress!
A few nits (that are my fault)
`It's` should be `its` in the first screen
For the second screen can we change it to
`Cluster installation has begun`
`This cluster has successfully been created and is in the installation process`
`To view the details of the installation process see the details page`

**Comment by matthewcarleton** [comment]
> > Cluster installation has begun
> > This cluster has successfully been created and is in the installation process
> > To view the details of the installation process see the details page
> 
> thats tricky since it might or might not have begun already. We just now it passed all validations and it is either in "ready" or "preparing for installation" or actually started installing. All we know that now it will start installing at some point :)

ok how about
`Cluster is ready`
`This cluster has been created and is ready to begin installation`
`To see the status of the installation see the details page`

**Comment by jelkosz** [comment]
this looks good to me!

**Comment by rawagner** [comment]
If I'd see `Cluster is ready` as title, I'd thought the cluster is ready to be used :) But whatever you prefer.

**Comment by matthewcarleton** [comment]
> If I'd see `Cluster is ready` as title, I'd thought the cluster is ready to be used :) But whatever you prefer.

we could say 
`Cluster is ready for installation`

**Comment by rawagner** [comment]
messages updated

File: src/cim/components/ClusterDeployment/ClusterDeploymentNetworkingStep.tsx [hunk 1]
```diff
@@ -87,7 +88,7 @@ const ClusterDeploymentNetworkingStep: React.FC<ClusterDeploymentDetailsNetworki
 
   const next = () => {
     // setCurrentStepId('something-next'); // TODO(mlibra): set the next step ID here
-    onClose(); // TODO(mlibra): just temporarily - the flow will continue
+    onFinish(); // TODO(mlibra): just temporarily - the flow will continue
```
Comment (mareklibra) [inline src/cim/components/ClusterDeployment/ClusterDeploymentNetworkingStep.tsx:91]: No review step???

**Review by mareklibra — APPROVED** [review]

**Pull request merged**